### PR TITLE
PARAMS: Update to Solo AC 3.5 params, new file for Solo AC 3.6 params

### PR DESCRIPTION
**Pull request by Pedals2Paddles**
- Disable logging pre-arm check in the for the Solo ArduCopter 3.5 params, since we still get random false positive logging failures.  ARMING_CHECK is now 7166.

- New param file for Solos running ArduCopter 3.6 since there are differences from 3.5

### CODE REVIEW

File: Tools/Frame_params/Solo_AC36.param [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+GND_EFFECT_COMP,1
+GPS_HDOP_GOOD,200
+GPS_NAVFILTER,6
+INS_ACC_BODYFIX,3
+INS_FAST_SAMPLE,3
+INS_TRIM_OPTION,2
+LAND_SPEED,45
+LAND_SPEED_HIGH,220
+LOG_BACKEND_TYPE,3
+LOG_DISARMED,1
```
Comment (muramura) [inline Tools/Frame_params/Solo_AC36.param]: I think that log output at disarmed time should be combined with Pixhawk.

File: Tools/Frame_params/Solo_AC36.param [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+COMPASS_ORIENT3,0
+COMPASS_PRIMARY,0
+COMPASS_TYPEMASK,15392
+COMPASS_USE,1
+COMPASS_USE2,1
+COMPASS_USE3,0
+DISARM_DELAY,8
+FENCE_ACTION,0
+FENCE_ALT_MAX,122
+FENCE_ENABLE,1
```
Comment (muramura) [inline Tools/Frame_params/Solo_AC36.param]: pixhawk's default is false.
Does it mean that it is changing intentionally?

File: Tools/Frame_params/Solo_AC36.param [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+COMPASS_ORIENT2,0
+COMPASS_ORIENT3,0
+COMPASS_PRIMARY,0
+COMPASS_TYPEMASK,15392
+COMPASS_USE,1
+COMPASS_USE2,1
+COMPASS_USE3,0
+DISARM_DELAY,8
+FENCE_ACTION,0
+FENCE_ALT_MAX,122
```
Comment (muramura) [inline Tools/Frame_params/Solo_AC36.param]: The pixhawk default is 100.
Does it mean that it is changing intentionally?

File: Tools/Frame_params/Solo_AC36.param (hunk 1 above)
Comment (Pedals2Paddles) [reply Tools/Frame_params/Solo_AC36.param]: Not sure what you mean?  The Solo has always had logging while disarmed enabled.

Comment (muramura) [reply Tools/Frame_params/Solo_AC36.param]: @Pedals2Paddles san. Comments Thaks.
That means you combined the original SOLO with the operation and settings.
I understand.

File: Tools/Frame_params/Solo_AC36.param (hunk 3 above)
Comment (Pedals2Paddles) [reply Tools/Frame_params/Solo_AC36.param]: Solo uses 122, which is 400FT.  Yes, intentional.

File: Tools/Frame_params/Solo_AC36.param (hunk 2 above)
Comment (Pedals2Paddles) [reply Tools/Frame_params/Solo_AC36.param]: Yes, intentional. Solo has this enabled by default.

File: Tools/Frame_params/Solo_AC36.param [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+SERVO1_FUNCTION,33
+SERVO2_FUNCTION,34
+SERVO3_FUNCTION,35
+SERVO4_FUNCTION,36
+THR_DZ,10
+VEL_XY_P,1.4
+WP_YAW_BEHAVIOR,3
+WPNAV_LOIT_MAXA,229
+WPNAV_LOIT_MINA,108
+WPNAV_SPEED,300
```
Comment (rmackay9) [inline Tools/Frame_params/Solo_AC36.param]: wpnav-speed of only 3m/s?  I'm happy if you're happy of course, but pretty slow!

**Comment by Pedals2Paddles** [comment]
Yep. It prevents the newbies who don't know what they're actually doing from rocketing off too fast.

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request closed without merging**